variables.tf

```
@@ -204,6 +204,24 @@ variable "storage_encrypted" {
   default     = false
 }
 
+variable "storage_type" {
+  type        = string
+  description = "One of 'standard' (magnetic), 'gp2' (general purpose SSD), or 'io1' (provisioned IOPS SSD)"
+  default     = "standard"
```

The docs do not mention standard nor gp2 and say the default is io1.

Are the docs out of date? Can we at least default this to null or io1 as it says? We dont want to break existing configs for anyone.

```suggestion
  default     = "io1"
```

---

yes the tf docs say the default is `io1`. I have updated this.